test/fn/create-users.spec.js

```diff
@@ -12,7 +12,9 @@ const userPrompt = rewire('../../src/lib/user-prompt');
 const readLine = require('readline-sync');
 const mockTestDir = testDir => sinon.stub(environment, 'pathToProject').get(() => testDir);
 
-describe('create-users', () => {
+describe('create-users', function () {
+  this.timeout(15000);
```

Comment:
I agree with you here, I was a bit disappointed to cause the tests to take so much longer (going from ~1 minute to ~7 minutes because of the repeated timeouts) but I think I've found a reasonable compromise.  
I went ahead and overrode `request` to directly all `rpn` in the related tests and I added a few tests to cover the retry mechanism in the api tests that will confidently tell us if we happen to break this feature